.azure/OneBranch.Publish.yml

```diff
@@ -15,6 +15,10 @@ variables:
   DisableDockerDetector: true
 
 parameters:
+- name: debug # debug mode will not actually upload and publish packages
```

Comment:
Is this automatically used somewhere?

Reply:
Line 119 uses it. I accidentally deleted it